web-wallet/src/lib/components/ContractOperations/ContractOperations.svelte

```diff
@@ -1,67 +1,70 @@
-<svelte:options immutable={true}/>
+<svelte:options immutable={true} />
 
 <script>
-	import { createEventDispatcher } from "svelte";
-	import {
-		mdiArrowDownBoldBoxOutline,
-		mdiArrowUpBoldBoxOutline,
-		mdiDatabaseArrowDownOutline,
-		mdiDatabaseOutline,
-		mdiHelpCircleOutline
-	} from "@mdi/js";
+  import { createEventDispatcher } from "svelte";
+  import {
+    mdiArrowDownBoldBoxOutline,
+    mdiArrowUpBoldBoxOutline,
+    mdiDatabaseArrowDownOutline,
+    mdiDatabaseOutline,
+    mdiHelpCircleOutline,
```

Comment:
similar issue here there is an extra comma that shouldn't be added

Reply:
Sadly this is how we decided to set up prettier.

Reply:
I see, but it only adds that extra comma if there are more than 2 items